packages/dd-trace/src/config.js

```diff
@@ -575,6 +580,7 @@ ken|consumer_?(?:id|key|secret)|sign(?:ed|ature)?|auth(?:entication|orization)?)
       (this.isIntelligentTestRunnerEnabled && !isFalse(DD_CIVISIBILITY_GIT_UPLOAD_ENABLED))
 
     this.gitMetadataEnabled = isTrue(DD_TRACE_GIT_METADATA_ENABLED)
+    this.isManualApiEnabled = this.isCiVisibility && isTrue(DD_CIVISIBILITY_MANUAL_API_ENABLED)
```

Comment:
I think that we should add the test in `config.spec.js`

Reply:
indeed, good catch

Reply:
added a test in https://github.com/DataDog/dd-trace-js/pull/3202/commits/d7768fa1687dd12d86033ddc978e219c154fafca